packages/bridge/src/navigation/navigation.ts

```diff
@@ -0,0 +1,14 @@
+import { v4 as uuidv4 } from "uuid";
+import { makeRequest } from "../messaging";
+import { BridgeMethod } from "../types";
+import { NavigateResult, NavigateRequest } from "./navigation.types";
+
+export function navigate(path: string) {
```

Comment:
I think for now the path is fine. There might be additional information (e.g. ids) one can send via url-parameters.

Reply:
Would you prefer a separate parameter to pass url params? Otherwise you could just pass something like `/my-app?param=123` as the path.

Reply:
That's what I thought. For now I do not see a need for extra params but maybe in the future.